tests/unit/mixins/route-test.js

```diff
@@ -308,3 +308,73 @@ test('it uses overridden params when loading more data', assert => {
   assert.ok(model.get('reachedInfinity'), 'Should reach infinity');
 
 });
+
+test('it uses bound params when loading more data', assert => {
+
+  assert.expect(11);
+
+  var RouteObject = Ember.Route.extend(RouteMixin, {
+    model() {
+      return this.infinityModel('item', {perPage: 1, startingPage: 1}, {category: 'feature'});
+    },
+
+    perPageParam: 'testPerPage',
+    pageParam: 'testPage',
+    totalPagesParam: 'meta.testTotalPages',
+    feature: Ember.computed.alias('test'),
+    test: 'new'
+  });
+  var route = RouteObject.create();
+
+  var dummyStore = {
+    find(name, params) {
+      assert.equal(route.get('test'), params.category, 'dynamic param is equal to the value of the computed property');
+      assert.ok(params.testPage);
```

Comment:
@kellyselden, apologies, I think you meant 321-323 and 332, will update the test shortly.